src/less/carousel/base/carousel.less

```diff
@@ -269,4 +273,19 @@ span.carousel__container {
         }
     }
     /* autoprefixer: on */
+    @media (max-width: 600px) {
+        .carousel {
+            &:not(&__autoplay) {
```

Comment:
Can you change `&__autoplay` to `.carousel__autoplay`.  I know its more verbose, but in this format we may miss this if someone does a search for all styles that apply to `carousel__autoplay`.

Reply:
> Also, based on the mocks, the icons don't quite look centered in the circles.

I changed a bit of the padding by 1px on this latest one, but can you be specific what doesn't look aligned. I tried to zoom in on the page and align them. They look fine to me now but let me know what part seems off (IE left/right/top/bottom alignment)